analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -137,11 +137,16 @@ class NuGetSupport(
             val packageData = registrationsBaseUrls.firstNotNullOfOrNull { getPackageData(id, it) }
 
             if (packageData == null) {
+                val hint = " Support for non-fixed versions, like '${id.version}', is not implemented".takeIf {
+                    !id.version.isFixedVersion()
+                }.orEmpty()
+
                 issues += createAndLogIssue(
                     source = "NuGet",
                     message = "Failed to get package data for '${id.toCoordinates()}' from any of " +
-                            "$registrationsBaseUrls."
+                            "$registrationsBaseUrls. $hint"
```

Comment:
The separating single space is already part of `hint` and thus should be omitted here.